packages/create-preact-app/lib/index.js

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env node
+const sade = require('sade');
+const notifier = require('update-notifier');
+const { error } = require('./util');
+const pkg = require('../package');
+
+const ver = process.version;
+const min = pkg.engines.node;
+if (
+	ver
+		.substring(1)
+		.localeCompare(min.match(/\d+/g).join('.'), 'en', { numeric: true }) === -1
+) {
+	return error(
+		`You are using Node ${ver} but create-preact-app requires Node ${min}. Please upgrade Node to continue!`,
+		1
+	);
+}
+
+// Safe to load async-based funcs
+const commands = require('./commands');
+
+// installHooks();
+notifier({ pkg }).notify();
+
+process.on('unhandledRejection', (err) => {
+	error(err.stack || err.message);
+});
+
+let prog = sade('create-preact-app').version(pkg.version);
+
+prog
+	.command('create [template] [dest]', '', { default: true })
```

Comment:
I made this the default so it can be ran as `npx create-preact-app default my-application`. I figure that's better than the alternative `npx create-preact-app create default my-application`.